pkg/operator/resources/batchapi/logging.go

```diff
@@ -43,17 +43,16 @@ func operatorLogStream(jobKey spec.JobKey) string {
 
 // Checks if log group exists before creating it
 func ensureLogGroupForAPI(apiName string) error {
-	output, err := config.AWS.CloudWatchLogs().DescribeLogGroups(&cloudwatchlogs.DescribeLogGroupsInput{
-		Limit:              aws.Int64(1),
-		LogGroupNamePrefix: aws.String(logGroupNameForAPI(apiName)),
-	})
+	apiNameLogGroup := logGroupNameForAPI(apiName)
+
+	logGroupExists, err := config.AWS.DoesLogGroupExist(apiNameLogGroup)
 	if err != nil {
 		return errors.WithStack(err)
```

Comment:
`WithStack` is not necessary here because any unwrapped error would have been wrapped in `DoesLogGroupExist`.

Reply:
Good catch. Thanks!